apps/editing-toolkit/editing-toolkit-plugin/whats-new/class-wp-rest-wpcom-whats-new-list.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * WP_REST_WPCOM_Whats_New_List file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+use Automattic\Jetpack\Connection\Client;
+
+/**
+ * Class WP_REST_WPCOM_Whats_New_List.
+ */
+class WP_REST_WPCOM_Whats_New_List extends \WP_REST_Controller {
+	/**
+	 * WP_REST_WPCOM_Block_Editor_Whats_New_Dot_Controller constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+	}
+
+	/**
+	 * Register available routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/whats-new/announcements',
```

Comment:
Wanted to re-suse route `whats-new/list` but whenever I use that I got an error saying route cannot be found. Is this because wpcom has the route already registered here `wp-content/rest-api-plugins/endpoints/whats-new.php`?